### ISSUE TITLE
Cannot configure boxroom to send email invitations through gmail

### ISSUE THREAD

**Issue by zaheenzkhan** [mischa78/boxroom#54]
Cannot configure boxroom to send email messages via gmail. Need help as i am not being able to add any users to groups

**Comment by mischa78** [comment]
I don't think this is a Boxroom issue. We're using Rails' standard ActionMailer. Please provide more details if you _do_ think this is an issue with Boxroom. Thank you.
